lib/QubitLftSyncer.class.php

```diff
@@ -0,0 +1,111 @@
+<?php
+
+class QubitLftSyncer
+{
+    private $parentId;
+    private $limit;
+
+    public function __construct($parentId, $limit = 10000)
```

Comment:
I'm not sure if users would end up manually sorting sets of children that large via the treeview so it might not be worth doing.

Reply:
I'm not sure I understand Mike. So, this is only needed for manual sorting in the treeview? One thing you didn't address is why is this needed and if we can fix the root of the issue, like in https://github.com/artefactual/atom/pull/1566. If this is only needed after a user moves a node in the tree we should not do it on each node load.

Reply:
Yeah, we're not sure about the actual root of the issue. Apparently even with #1566 we're still running into the issue so it could be that there's another issue causing Elasticsearch's lft values to "desync" with the values in the database (or, given that ES updates asynchronously, ES might not be updating quick enough to keep in sync with the DB changes made while a user moves nodes).

Great idea, though, to have the desync check/repair only happen when reloaded treeview nodes after a move. I'll look into making that change.